build.gradle

```diff
@@ -538,8 +538,12 @@ def sonarExclusions = [
   '**uk/gov/hmcts/reform/prl/config/**/*',
   '**uk/gov/hmcts/reform/prl/utils/*',
   '**uk/gov/hmcts/reform/prl/Application.*',
-  '**uk/gov/hmcts/reform/prl/controllers/citizen/CaseController.java',
-  '**uk/gov/hmcts/reform/prl/controllers/citizen/CitizenCallbackController.java'
+  '**uk/gov/hmcts/reform/prl/controllers/citizen/mapper/CaseDataMapper.java',
```

Comment:
Can we please remove these exclusions?

Reply:
will be covered https://tools.hmcts.net/jira/browse/PRLC100-429